src/utils/detect-server-settings.mjs

```diff
@@ -361,14 +376,17 @@ const detectServerSettings = async (devConfig, options, projectDir, metadata) =>
   /** @type {Partial<import('./types.js').BaseServerSettings>} */
   let settings = {}
 
+  const fs = new NodeFS()
+  const project = new Project(fs, projectDir)
```

Comment:
This should be already available on the base command where we use that to get the analytics for frameworks.

@sarahetter did that.

Reply:
good idea! implmented in https://github.com/netlify/cli/pull/5816/commits/1408884f932620acb5f4662d8404c80b2362d9b7